src/test/java/me/karakelley/http/server/HttpServerTest.java

```diff
@@ -45,7 +42,7 @@ void testSendsResponseWithHeadersGivenValidRequest() throws Exception {
     ClientHelper client = new ClientHelper();
     ServerConfiguration config = new ServerConfiguration();
     config.setPort("0");
-    config.setHandler(new Application());
+    config.setHandler(new ApplicationFactory().create("nodirectory"));
```

Comment:
I feel like there have been multiple PRs now where every test has needed the same small tweak. Is there any logic that can be extracted into `private` methods?

Reply:
I've been wanting to do that for a while but I thought moving all the object instantiation to each individual test was to help with visibility? At what point is it ok to extract some duplication to a method? Is visibility preferred over duplication?